app/lib/actions/katello/capsule_content/sync.rb

```diff
@@ -39,9 +39,11 @@ def plan(smart_proxy, options = {})
           refresh_options = options.merge(content_view: content_view,
                                              environment:  environment,
                                              repository: repository)
-          plan_action(Actions::Pulp::Orchestration::Repository::RefreshRepos, smart_proxy, refresh_options)
-          plan_action(Actions::Pulp3::Orchestration::Repository::RefreshRepos, smart_proxy, refresh_options)
-          plan_action(SyncCapsule, smart_proxy, refresh_options)
+          sequence do
+            plan_action(Actions::Pulp::Orchestration::Repository::RefreshRepos, smart_proxy, refresh_options)
+            plan_action(Actions::Pulp3::Orchestration::Repository::RefreshRepos, smart_proxy, refresh_options)
+            plan_action(SyncCapsule, smart_proxy, refresh_options)
+          end
         end
```

Comment:
Yayyy! Fixed all issues.. :smile: Will run through use-cases on the PR. Meanwhile, I see the repo, version and the contents so passes smoke test. :+1: